Author: Peter Chiu <chui-hao.chiu@mediatek.com>
Date:   Tue Oct 17 16:32:51 2023 +0800

    wifi: mt76: mt7915: update mt798x_wmac_adie_patch_7976
    
    Add support for a newer ADIE version
    
    Signed-off-by: Peter Chiu <chui-hao.chiu@mediatek.com>
    Signed-off-by: Felix Fietkau <nbd@nbd.name>
---

--- a/mt7915/soc.c
+++ b/mt7915/soc.c
@@ -517,7 +517,8 @@
 	if (ret)
 		return ret;
 
-	if (version == 0x8a00 || version == 0x8a10 || version == 0x8b00) {
+	if (version == 0x8a00 || version == 0x8a10 ||
+	    version == 0x8b00 || version == 0x8c10) {
 		rg_xo_01 = 0x1d59080f;
 		rg_xo_03 = 0x34c00fe0;
 	} else {

